pkg/kubelet/lifecycle/admission_failure_handler_stub.go

```diff
@@ -31,6 +30,6 @@ func NewAdmissionFailureHandlerStub() *AdmissionFailureHandlerStub {
 	return &AdmissionFailureHandlerStub{}
 }
 
-func (n *AdmissionFailureHandlerStub) HandleAdmissionFailure(admitPod *v1.Pod, failureReasons []predicates.PredicateFailureReason) (bool, []predicates.PredicateFailureReason, error) {
+func (n *AdmissionFailureHandlerStub) HandleAdmissionFailure(admitPod *v1.Pod, failureReasons []PredicateFailureReason) (bool, []PredicateFailureReason, error) {
```

Comment:
Would it cause compatibility issues if we update the interface here? Would someone writer their custom admission failure handler which out of the repo?

Reply:
Nope. Explained above.